minifi-assembly/NOTICE

```diff
@@ -12,15 +12,35 @@ The following binary components are provided under the Apache Software License v
 
   (ASLv2) Apache NiFi
     The following NOTICE information applies:
-      Apache NiFi
-      Copyright 2014-2016 The Apache Software Foundation
+        Apache NiFi
+        Copyright 2014-2016 The Apache Software Foundation
+
+        This product includes software developed at
+        The Apache Software Foundation (http://www.apache.org/).
+
+        This product includes the following work from the Apache Hadoop project:
+
+        BoundedByteArrayOutputStream.java adapted to SoftLimitBoundedByteArrayOutputStream.java
+
+        This includes derived works from the Apache Software License V2 library python-evtx (https://github.com/williballenthin/python-evtx)
+        Copyright 2012, 2013 Willi Ballenthin william.ballenthin@mandiant.com
+        while at Mandiant http://www.mandiant.com
+        The derived work is adapted from Evtx/Evtx.py, Evtx/BinaryParser.py, Evtx/Nodes.py, Evtx/Views.py and can be found in the org.apache.nifi.processors.evtx.parser package.
 
-      This product includes software developed at
-      The Apache Software Foundation (http://www.apache.org/).
 
-      This product includes the following work from the Apache Hadoop project:
 
-      BoundedByteArrayOutputStream.java adapted to SoftLimitBoundedByteArrayOutputStream.java
+        This includes derived works from the Apache Storm (ASLv2 licensed) project (https://github.com/apache/storm):
```

Comment:
I believe this, Evtx and the Hive items can be removed as we do not have these items included as those NARs are not currently bundled.